src/components/Documents/DocumentTable.jsx

```diff
@@ -32,47 +32,142 @@ import { EmptyListNotification, LoadingAnimation } from '../Notification';
  * @returns {React.JSX.Element} The DocumentTable component
  */
 const DocumentTable = ({ handleAclPermissionsModal, handleSelectDeleteDoc }) => {
+  const { session } = useSession();
   const { documentListObject, loadingDocuments } = useContext(DocumentListContext);
+
+  const handleShowDocumentLocal = async (urlToOpen) => {
+    const urlFileBlob = await getBlobFromSolid(session, urlToOpen);
+    window.open(urlFileBlob);
+  };
+
   const columnTitlesArray = [
-    'Name',
-    'Type',
-    'Description',
-    'Upload Date',
-    'Expiration Date',
-    'Preview File',
-    'Sharing',
-    'Delete'
+    {
+      headerName: 'Name',
+      field: 'name',
+      minWidth: 120,
+      flex: 1,
+      headerAlign: 'center',
+      align: 'center'
+    },
+    {
+      headerName: 'Type',
+      field: 'type',
+      minWidth: 120,
+      flex: 1,
+      headerAlign: 'center',
+      align: 'center'
+    },
+    {
+      headerName: 'Description',
+      field: 'description',
+      minWidth: 120,
+      flex: 1,
+      headerAlign: 'center',
+      align: 'center'
+    },
+    {
+      field: 'upload date',
+      type: 'dateTime',
+      valueGetter: ({ value }) => value && new Date(value),
```

Comment:
Think we can adjust the date objects into strings. Just the date would be fine. The same for Line 80 as well.

Reply:
This type of column throws an error when the date is converted to a string. I can remove the type `type: 'dateTime'`. Here is a screen render the differences:
![image](https://github.com/codeforpdx/PASS/assets/96048/e7db80b8-9374-4767-bc03-055d3a9d8b75)

Reply:
Was thinking along the lines of `toLocaleDateString`